src/create-ad-slot.ts

```diff
@@ -148,28 +148,21 @@ const createClasses = (
  */
 const concatSizeMappings = (
```

Comment:
It would be good to have some tests around this breakpoint behaviour if possible

Reply:
One of the existing tests covers rejecting invalid keys (see the `invalid` mapping in the test), but I'll add some tests around mutating the original size mappings.

Reply:
Actually I may delay this to a follow-up PR, I thought it'd be easy but looks like something isn't quite right with the ad sizes when running Jest. For example, running the following gives me `TypeError: undefined is not a function` rather than a proper diff of the expected/actual

```typescript
expect({ desktop: [adSizes.mobilesticky] }).toEqual({
	desktop: [adSizes.googleCard],
});
```